help/include/when-to-start-a-new-topic.md

```diff
@@ -2,7 +2,7 @@ To get the full benefits of Zulip's topic model, when starting a new
 conversation, you should start a new topic!
 
 Starting a topic is like a lighter weight version of giving your email a subject.
-Topic names should be brief but specific, e.g.:
+Topic names should be brief but specific, e.g.,:
```

Comment:
I think `.,:` at the end of a line is kinda ugly; we might want to convert that pattern into "for example:" in a follow-up pass.

Reply:
Yeah, I agree and thought about it when making these changes. A follow-up  pass makes sense to me.